contracts/SSVNetwork.sol

```diff
@@ -96,7 +96,7 @@ contract SSVNetwork is OwnableUpgradeable, ISSVNetwork {
     uint64 private _declareOperatorFeePeriod;
     uint64 private _executeOperatorFeePeriod;
     uint64 private _operatorMaxFeeIncrease;
-
+    uint64 private _minimumBlocksBeforeLiquidation;
```

Comment:
@vadiminc maybe you thought of this before but I'd rather ask. I see the purpose of this variable, and checking `_liquidatable` function that applies to pods, would it be possible to compute on the fly this `_minimumBlocksBeforeLiquidation` using operators' `_declareOperatorFeePeriod` and `_executeOperatorFeePeriod` ? I know they are measured in different units (blocks and timestamp). Just an idea. Thanks!

Reply:
@mtabasco thats ok, you can ask any questions ;) _minimumBlocksBeforeLiquidation, _declareOperatorFeePeriod and  _executeOperatorFeePeriod are not connected. _minimumBlocksBeforeLiquidation - says for how many bocks you should have tokens on your balance at least.  FeePeriod - about the operator fee changes flow.